plugins/inputs/mock/mock_test.go

```diff
@@ -0,0 +1,80 @@
+package mock
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please only use `require` and avoid using `assert`.

Reply:
ah yes, thx